Makefile

```diff
@@ -29,7 +31,6 @@ image-build:
 		--build-arg PKG=$(PKG) \
 		--build-arg SRC=$(SRC) \
 		--build-arg TAG=$(TAG:$(BUILD_META)=) \
-		--build-arg ARCH=$(ARCH) \
```

Comment:
If we don't pass the ARCH, then it takes the ARCH of the building host?

Reply:
no we use TARGETARCH instead which is filled automatically by docker